package.json

```diff
@@ -27,7 +27,16 @@
         "gem": "^2.4.3"
     },
     "scripts": {
-        "test": "jest"
+        "test": "jest",
+        "debug": "grunt debug",
```

Comment:
Added scripts to call the grunt tasks and to run various web-ext tasks for quick debugging.